src/components/organisms/Warsaw/Content.jsx

```diff
@@ -6,11 +6,18 @@ import imagePathWebp700 from './office_700.webp';
 import imagePathWebp1400 from './office_1400.webp';
 import imagePathJpg700 from './office_700.jpg';
 import imagePathJpg1400 from './office_1400.jpg';
+import Section from '../../molecules/Section';
 
-const Wrapper = styled.div`
+const Row = styled.div`
   display: flex;
+  margin: -1em -1em 1em -1em;
   flex-wrap: wrap;
-  margin: -1em;
+`;
+
+const TextWrapper = styled.div`
+  flex: 1 0 20em;
+  max-width: calc(100% - 2em);
+  margin: 1em;
```

Comment:
Looks like there is no need for this changes. They do nothing but renaming and unwanted changing margins.